src/services/dashboards/dashboard-detail/modules/DashboardWidgetContainer.vue

```diff
@@ -132,7 +132,7 @@ export default defineComponent<Props>({
             if (state.initiatedWidgetMap[target.id]) return;
             if (isIntersecting) {
                 const targetWidgetRef: WidgetComponent|null = state.widgetRef.find((d) => d?.$el?.id === target.id);
-                if (typeof targetWidgetRef?.initWidget === 'function') {
+                if (typeof targetWidgetRef?.initWidget === 'function' && dashboardDetailValidationState.widgetValidMap[target.id] !== false) {
```

Comment:
```suggestion
                if (typeof targetWidgetRef?.initWidget === 'function' && dashboardDetailValidationState.widgetValidMap[target.id] === true) {
```
Looks like `widgetValidMap` could be `undefined`. How about this ?!

Reply:
If the widget is valid, it may not be in the `widgetValidMap`..

Reply:
Oh .. got it